Makefile

```diff
@@ -222,26 +222,31 @@ clean_certs:
 ###                  Formatting, linting, and vetting                       ###
 ###############################################################################
 
-format:
-	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
-	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/cometbft/cometbft
-.PHONY: format
+# https://github.com/cometbft/cometbft/pull/1925#issuecomment-1875127862
+# Revisit using format after CometBFT v1 release and/or after 2024-06-01.
+#format:
+#	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
+#	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/cometbft/cometbft
+#.PHONY: format
 
 lint:
 	@echo "--> Running linter"
 	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run
 .PHONY: lint
 
-lint-format:
-	@go run github.com/golangci/golangci-lint/cmd/golangci-lint@latest run --fix
-	@go run mvdan.cc/gofumpt -l -w ./..
-.PHONY: lint-format
+# https://github.com/cometbft/cometbft/pull/1925#issuecomment-1875127862
+# Revisit using lint-format after CometBFT v1 release and/or after 2024-06-01.
+#lint-format:
```

Comment:
I don't think we should comment these commands. The current `main` branch passes them.

Reply:
I'm in agreement here, because I find that linting helps enormously with keeping code legible and frequently catches a surprising number of issues that are easy for humans to miss.